plugins/ComposeAuthUI/src/commonMain/kotlin/io/github/jan/supabase/compose/auth/ui/AuthState.kt

```diff
@@ -65,6 +65,7 @@ class AuthState(
 /**
  * Local composition for [AuthState]. Use [AuthForm] for automatic saving and restoring.
  */
+@AuthUiExperimental
```

Comment:
Is this one needed?

Reply:
Yes, pretty much all the features are experimental, as I haven't updated or improved the UI library further since the first release.